naga/src/front/wgsl/error.rs

```diff
@@ -899,6 +919,55 @@ impl<'a> Error<'a> {
                 )
                 .into()],
             },
+            Error::EnableExtensionNotYetImplemented { kind, span } => ParseError {
+                message: format!(
+                    "the `{}` extension is not yet supported (sorry!)",
```

Comment:
suggestion: We probably don't need the `the ` article or the ` (sorry!)` parenthetical; it's generally more interesting to keep the opening description of the error succinct, IME.